antlr/YorkieSchemaParser.ts

```
@@ -0,0 +1,2200 @@
+	}
+
+}
+
+export class DocumentContext extends ParserRuleContext {
+	public declarationList(): DeclarationListContext {
+		return this.getRuleContext(0, DeclarationListContext);
+	}
+	public EOF(): TerminalNode { return this.getToken(YorkieSchemaParser.EOF, 0); }
+	constructor(parent: ParserRuleContext | undefined, invokingState: number) {
+		super(parent, invokingState);
+	}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)